src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
The Random object should be a private class-level field instead of being instantiated every time when 'getRandomBall' method is called. This is because creating a new Random instance each time may not be efficient and could lead to less randomness in the generated numbers if the calls are close in time.